app/views/courses/_aside-filters.html.erb

```diff
@@ -1,22 +1,27 @@
 <% unless @course_filter.nil? %>
-<h2 class="govuk-heading-s">Filter courses</h2>
+<div class="ncce-courses__filters-wrapper__title-desktop">
```

Comment:
Its not usual in the BEM model to have two lots of __, I would suggest it could be 
`ncce-courses__filter-wrapper-title--desktop`
and
`ncce-courses__filter-wrapper-title--mobile`
as the last bit is more of a modifier